<launch>
  <arg name="video_device" default="/dev/video0" />
  <arg name="camera_info_url" default="usb_camera_720p_8.yaml" />
  <node name="lolo_camera" pkg="usb_cam" type="usb_cam_node" >
    <param name="video_device" value="$(arg video_device)" />
    <param name="camera_info_url" value="file://$(find lolo_perception)/config/camera_calibration_data/$(arg camera_info_url)" />
    <param name="image_width" value="1280" /> <!--640-->
    <param name="image_height" value="720" /> <!--480-->
    <param name="pixel_format" value="mjpeg" /> <!--yuyv-->
    <param name="camera_frame_id" value="lolo_camera" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="image_proc" pkg="image_proc" type="image_proc" ns="lolo_camera">
    <remap from="image" to="/lolo_camera/image_raw"/>
  </node>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/lolo_camera/image_rect_color"/>
    <param name="autosize" value="true" />
  </node>
</launch>
